src/universal/modules/meeting/components/MeetingHelpDialog/MeetingHelpDialogMenu.js

```diff
@@ -46,8 +48,9 @@ const DialogClose = styled(FontAwesome)({
 })
 
 const MeetingHelpDialogMenu = (props: Props) => {
-  const {closePortal, phase} = props
+  const {closePortal, meetingType, phase} = props
   const phaseLabel = phaseLabelLookup[phase]
+  const phaseHelpLookup = meetingType === ACTION ? actionPhaseHelpLookup : retroPhaseHelpLookup
```

Comment:
it looks like we're starting to store a bunch of components in lookup tables.
that isn't ideal because i should only have to load the tip that i want to view, not every tip for every meeting type.
Instead, can we use react-loadable components like we do for async menus?